include/envoy/json/json_object.h

```diff
@@ -85,6 +85,18 @@ class Object {
    */
   virtual bool isNull() const PURE;
 
+  /**
```

Comment:
these shouldn't appear in diff, can you merge master?

Reply:
@lizan done thanks!